tests/FSharp.Core.UnitTests/FSharp.Core/Microsoft.FSharp.Collections/ArrayModule.fs

```diff
@@ -1242,6 +1242,77 @@ type ArrayModule() =
         CheckThrowsArgumentNullException(fun () -> Array.groupBy funcInt (null : int array) |> ignore)
         ()
 
+    [<Fact>]
```

Comment:
Honestly, the test is a bit hard to follow, I'd prefer having it split into parts (e.g. by each "compareAndAssert" statement)

Reply:
I chose consistency with this particular test suite (pretty much 1 function per collection function).